webapp/src/Twig/TwigExtension.php

```diff
@@ -965,4 +966,20 @@ public function hexColorToRGBA(string $text, float $opacity = 1): string
 
         return $text;
     }
+
+    /**
+     * Convert the given string to a field that is safe to use in a TSV file
+     *
+     * @param string $field
+     *
+     * @return string
+     */
+    public function toTsvField(string $field)
+    {
+        return str_replace(
+            ["\\", "\t", "\n", "\r"],
```

Comment:
Do you use double vs single ticks (below) intentionally? Also you might want to align the list elements.